LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Algo 2 went MIT and I didn't see a reason why we shouldn't as well. We can discuss this at standup today if anyone has concerns.